### DISCUSSION TITLE
alchemy: useBalance watch

### DISCUSSION THREAD

**Discussion by callmejm** [wevm/wagmi#1349] [Q&A]
Hi, I would like to ask, should I be worry wasting alchemy compute unit, if by using `useBalance`  for specific token , and params `watch` set to `true`.

**Comment by danilo-timacum** [comment]
Also interested in this question.
Is it possible to fetch balance data directly from wallet extension (eg. Metamask) instead sending requests to network each time?

**Comment by huytq03** [comment]
I have the same issue.
